DOCSP-37540/1/9 Limit, Sort, Skip (#66)
mongodb/docs-pymongo: source/read/specify-documents-to-return.txt

      :end-before: end-skip-option
      :language: python

   .. output::

      Dorrian'S Red Hand Restaurant
      The Princeton Club
      Moran'S Chelsea
      La Parisienne Diner
      Jimmy'S Corner
      ...

Combine Limit, Sort, and Skip
-----------------------------

You can combine the ``limit()``, ``sort()``, and ``skip()`` methods in a single
operation. This allows you to set a maximum number of sorted documents to
return, skipping a specified number of documents before returning.

The following example returns documents with the ``cuisine`` value of
``"Italian"``. The results are sorted in alphabetical order, skipping the first
10 documents:

.. io-code-block::

   .. input:: /includes/read/specify-documents-to-return.py
      :start-after: start-limit-sort-skip
      :end-before: end-limit-sort-skip
      :language: python

   .. output::
      
      Acqua
      Acqua Restaurant
      Acqua Santa
      Acquista Trattoria
      Acquolina Catering

.. note::

   The order in which you call these methods doesn't change the documents
   that are returned. The driver automatically reorders the calls to apply the
   sort and skip first and the limit after.

> You can also apply ``limit``, ``sort`` and ``skip`` by specifying them as

Author: Pretty sure my brain must have been translating backticks into commas in this PR

> You can also apply ``limit``, ``sort`` and ``skip`` by specifying them as
> options in the ``find()`` method. The following example specifies the

Reviewer: S:

```suggestion
You can also limit, sort, and skip results by using parameters of
the ``find()`` method. The following example specifies the
```
